src/ripple/nodestore/impl/Shard.cpp

```diff
@@ -583,17 +583,6 @@ Shard::finalize(
     {
         state_ = finalizing;
 
-        /*
-        TODO MP
-        A lock is required when calling the NuDB verify function. Because
-        this can be a time consuming process, the server may desync.
-        Until this function is modified to work on an open database, we
-        are unable to use it from rippled.
-
-        // Verify backend integrity
-        backend_->verify();
-        */
```

Comment:
Let's check with @miguelportilla before removing this and the `verify` implementation.

Reply:
I am okay removing this commented out code. Although the backend verify function isn't used, it could be useful at some point. For instance, the above situation can be addressed by calling verify on the new deterministic backend. I could also see a server startup switch to have the backend be verified at startup.